components/channel_header/index.js

```diff
@@ -47,12 +47,14 @@ import {isFileAttachmentsEnabled} from 'utils/file_utils';
 
 import ChannelHeader from './channel_header';
 
+const EMPTY_CHANNEL = {};
+
 function makeMapStateToProps() {
     const doGetProfilesInChannel = makeGetProfilesInChannel();
     const getCustomStatus = makeGetCustomStatus();
 
     return function mapStateToProps(state) {
-        const channel = getCurrentChannel(state) || {};
+        const channel = getCurrentChannel(state) || EMPTY_CHANNEL;
```

Comment:
regarding my comment above this is a unneeded change

Reply:
https://github.com/mattermost/mattermost-webapp/pull/9808#discussion_r807647580